Firestore/Source/API/FIRFirestoreSettings.mm

```diff
@@ -53,22 +55,19 @@ - (BOOL)isEqual:(id)other {
          self.isSSLEnabled == otherSettings.isSSLEnabled &&
          self.dispatchQueue == otherSettings.dispatchQueue &&
          self.isPersistenceEnabled == otherSettings.isPersistenceEnabled &&
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-         self.timestampsInSnapshotsEnabled == otherSettings.timestampsInSnapshotsEnabled &&
-#pragma clang diagnostic pop
-         self.cacheSizeBytes == otherSettings.cacheSizeBytes;
+         SUPPRESS_DEPRECATED_DECLARATIONS_BEGIN()
```

Comment:
I think you could add a trailing semicolon here to make clang-format behave.

The macros are set up to just contain the pragmas without semicolons in order to make them usable outside a statement block. Within a block extra semicolons should be harmless.